skywalking-sniffer/skywalking-toolkit-activation/skywalking-toolkit-opentracing-activation/src/main/java/com/a/eye/skywalking/toolkit/activation/opentracing/span/SkyWalkingSpanActivation.java

```diff
@@ -0,0 +1,80 @@
+package com.a.eye.skywalking.toolkit.activation.opentracing.span;
+
+import com.a.eye.skywalking.plugin.interceptor.ConstructorInterceptPoint;
+import com.a.eye.skywalking.plugin.interceptor.InstanceMethodsInterceptPoint;
+import com.a.eye.skywalking.plugin.interceptor.enhance.ClassInstanceMethodsEnhancePluginDefine;
+
+import net.bytebuddy.description.method.MethodDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+
+import java.util.Map;
+
+import static net.bytebuddy.matcher.ElementMatchers.named;
+import static net.bytebuddy.matcher.ElementMatchers.noneOf;
+import static net.bytebuddy.matcher.ElementMatchers.takesArguments;
+
+/**
+ * Created by xin on 2017/1/16.
+ */
+public class SkyWalkingSpanActivation extends ClassInstanceMethodsEnhancePluginDefine {
+    @Override
+    protected String enhanceClassName() {
+        return "com.a.eye.skywalking.toolkit.opentracing.SkywalkingSpan";
+    }
+
+    @Override
+    protected ConstructorInterceptPoint[] getConstructorsInterceptPoints() {
+        return new ConstructorInterceptPoint[]{
+                new ConstructorInterceptPoint() {
+                    @Override
+                    public ElementMatcher<MethodDescription> getConstructorMatcher() {
+                        return takesArguments(String.class, long.class, Map.class);
+                    }
+
+                    @Override
+                    public String getConstructorInterceptor() {
+                        return "com.a.eye.skywalking.toolkit.activation.opentracing.span.interceptor.SpanConstructorInterceptor";
```

Comment:
Not the correct name again. No need of `Span` prefix, it already existed in package name.
Maybe `com.a.eye.skywalking.toolkit.activation.opentracing.span.interceptor.Constructor`? or use other principles.